src/applications/personalization/profile-2/components/direct-deposit/DirectDepositContent.jsx

```diff
@@ -0,0 +1,311 @@
+import React, { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import ReactCSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import EbenefitsLink from 'platform/site-wide/ebenefits/containers/EbenefitsLink';
+import { isLOA3, isMultifactorEnabled } from 'platform/user/selectors';
+
+import {
+  editModalToggled,
+  savePaymentInformation as savePaymentInformationAction,
+} from 'applications/personalization/profile360/actions/paymentInformation';
+import {
+  directDepositAccountInformation,
+  directDepositInformation,
+  directDepositIsSetUp,
+  directDepositUiState as directDepositUiStateSelector,
+} from 'applications/personalization/profile360/selectors';
+
+import BankInfoForm from 'applications/personalization/profile360/components/BankInfoForm';
+import PaymentInformationEditError from 'applications/personalization/profile360/components/PaymentInformationEditModalError';
+
+import ProfileInfoTable from '../ProfileInfoTable';
+import FraudVictimAlert from './FraudVictimAlert';
+import AdditionalInformation from './DirectDepositInformation';
+
+import { prefixUtilityClasses } from '../../helpers';
+
+// TODO: move this to a common location (in platform?) for easy sharing
```

Comment:
Do we still want to do this or can we remove this comment? :)

Reply:
yeah, I want to move it after this gets merged. thanks for the reminder!